grafana/dashboards/cmode/harvest_dashboard_quotaReport.json

```diff
@@ -54,11 +54,12 @@
   "gnetId": null,
   "graphTooltip": 0,
   "id": null,
-  "iteration": 1660910569138,
+  "iteration": 1662734055395,
   "links": [],
   "panels": [
     {
       "datasource": "${DS_PROMETHEUS}",
+      "description": "Only tree quotas are shown by default. To collect  user/group quotas, enable in qtree.yaml template",
```

Comment:
Maybe
"Only tree quotas are shown by default. To collect user/group quotas, enable them in the qtree.yaml template"